### PR TITLE
MCR-1815 check pi exist direct after generation

### PR DESCRIPTION
**Pull request by sebhofmann**

### CODE REVIEW

File: mycore-pi/src/main/resources/components/pi/config/messages_de.properties [hunk 1]
```diff
@@ -4,6 +4,7 @@ component.pi.register.dnbUrn.success=URN Registrierung erfolgreich.
 component.pi.register.doi=DOI Registrieren
 component.pi.register.doi.error=DOI Registrierung fehlgeschlagen.
 component.pi.register.doi.success=DOI Registrierung erfolgreich.
+component.pi.register.error.1=Der generierte Identifikator  existiert berreits in der Datenbank.
```
Comment (kkrebs) [inline mycore-pi/src/main/resources/components/pi/config/messages_de.properties]: fixed typos:
Der generierte Identifikator existiert bereits in der Datenbank.

File: mycore-pi/src/main/resources/components/pi/config/messages_en.properties [hunk 2]
```diff
@@ -4,6 +4,7 @@ component.pi.register.dnbUrn.success=URN registration successfully.
 component.pi.register.doi=register doi
 component.pi.register.doi.error=DOI registration failed.
 component.pi.register.doi.success=DOI registration successfully.
+component.pi.register.error.1=The Generated identifier is already present in database.
```
Comment (kkrebs) [inline mycore-pi/src/main/resources/components/pi/config/messages_en.properties]: fixed typo:
The generated identifier is already present in database.

File: mycore-pi/src/main/java/org/mycore/pi/MCRPIRegistrationService.java [hunk 3]
```diff
@@ -32,12 +30,15 @@
...
 import org.mycore.pi.backend.MCRPI;
 import org.mycore.pi.exceptions.MCRPersistentIdentifierException;
+import org.mycore.services.i18n.MCRTranslation;
 
 import com.google.gson.ExclusionStrategy;
 import com.google.gson.FieldAttributes;
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 
+import static org.mycore.access.MCRAccessManager.PERMISSION_WRITE;
```
Comment (rsteph-de) [inline mycore-pi/src/main/java/org/mycore/pi/MCRPIRegistrationService.java:40]: I would avoid static imports and use class name + variable (MCRAccessManager.PERMISSION_WRITE) where it is used

**Review by rsteph-de — COMMENTED** [review]
line 195: setting values to variables in method parameters creates code, that is difficult to read.

**Review by kkrebs — APPROVED** [review]

**Pull request merged**